index.html

```diff
@@ -137,7 +137,7 @@
 
     ### Use smart distribution
 
-    Choose a platform on which to publish your polyfill from which developers can easily consume it in a way that keeps up to date with changes to the original source.  For example, use a module registry like [npmjs.org](https://npmjs.org/), and consider hosting the polyfill on a service from which it can be directly loaded in the browser, such as [cdnjs.com](https://cdnjs.com/) or [polyfill.io](https://polyfill.io/).
+    Choose a platform on which to publish your polyfill from which developers can easily consume it in a way that keeps up to date with changes to the original source.  For example, use a module registry like [npmjs.org](https://npmjs.org/), and consider hosting the polyfill on a reputable service from which it can be directly loaded in the browser.
```

Comment:
```suggestion
    Choose a platform on which to publish your polyfill from which developers can easily consume it in a way that keeps up to date with changes to the original source.  For example, use a module registry and consider hosting the polyfill on a reputable service from which it can be directly loaded in the browser.
```

Reply:
Channeling comment from @ylafon: "we could also remove links to cdnjs and npmjs because... they can break also, and there is no real need to link to them."